configs/swarm/genesis.json

```diff
@@ -1,191 +1,190 @@
-      },
-      {
-        "NewParameter": "?WSVAssetMetadataLimits=1048576,4096_ML"
-      },
-      {
-        "NewParameter": "?WSVTriggerMetadataLimits=1048576,4096_ML"
-      },
-      {
-        "NewParameter": "?WSVIdentLengthLimits=1,128_LL"
-      },
-      {
-        "NewParameter": "?ExecutorFuelLimit=55000000"
-      },
-      {
-        "NewParameter": "?ExecutorMaxMemory=524288000"
-      },
-      {
-        "NewParameter": "?WASMFuelLimit=55000000"
-      },
-      {
-        "NewParameter": "?WASMMaxMemory=524288000"
-      },
-      {
-        "Register": {
-          "Role": {
-            "id": "ALICE_METADATA_ACCESS",
-            "permissions": [
-              {
-                "id": "CanRemoveKeyValueInAccount",
-                "payload": {
-                  "account_id": "ed0120CE7FA46C9DCE7EA4B125E2E36BDB63EA33073E7590AC92816AE1E861B7048B03@wonderland"
-                }
-              },
-              {
-                "id": "CanSetKeyValueInAccount",
-                "payload": {
-                  "account_id": "ed0120CE7FA46C9DCE7EA4B125E2E36BDB63EA33073E7590AC92816AE1E861B7048B03@wonderland"
-                }
+      }
+    },
+    {
+      "NewParameter": "?MaxTransactionsInBlock=512"
+    },
+    {
+      "NewParameter": "?BlockTime=2000"
+    },
+    {
+      "NewParameter": "?CommitTimeLimit=4000"
+    },
+    {
+      "NewParameter": "?TransactionLimits=4096,4194304_TL"
+    },
+    {
+      "NewParameter": "?WSVDomainMetadataLimits=1048576,4096_ML"
+    },
+    {
+      "NewParameter": "?WSVAssetDefinitionMetadataLimits=1048576,4096_ML"
+    },
+    {
+      "NewParameter": "?WSVAccountMetadataLimits=1048576,4096_ML"
+    },
+    {
+      "NewParameter": "?WSVAssetMetadataLimits=1048576,4096_ML"
+    },
+    {
+      "NewParameter": "?WSVTriggerMetadataLimits=1048576,4096_ML"
+    },
+    {
+      "NewParameter": "?WSVIdentLengthLimits=1,128_LL"
+    },
+    {
+      "NewParameter": "?ExecutorFuelLimit=55000000"
+    },
+    {
+      "NewParameter": "?ExecutorMaxMemory=524288000"
+    },
+    {
+      "NewParameter": "?WASMFuelLimit=55000000"
+    },
+    {
+      "NewParameter": "?WASMMaxMemory=524288000"
+    },
+    {
+      "Register": {
+        "Role": {
+          "id": "ALICE_METADATA_ACCESS",
+          "permissions": [
+            {
+              "id": "CanRemoveKeyValueInAccount",
+              "payload": {
+                "account_id": "ed0120CE7FA46C9DCE7EA4B125E2E36BDB63EA33073E7590AC92816AE1E861B7048B03@wonderland"
               }
-            ]
-          }
+            },
+            {
+              "id": "CanSetKeyValueInAccount",
+              "payload": {
+                "account_id": "ed0120CE7FA46C9DCE7EA4B125E2E36BDB63EA33073E7590AC92816AE1E861B7048B03@wonderland"
+              }
+            }
+          ]
         }
       }
-    ]
+    }
   ],
-  "executor_file": "./executor.wasm"
+  "executor_file": "./executor.wasm",
+  "chain_id": "0"
```

Comment:
```zsh
$ docker-compose -f configs/swarm/docker-compose.local.yml up

irohad0-1  | thread 'sumeragi thread' panicked at core/src/sumeragi/main_loop.rs:280:14:
irohad0-1  | Genesis invalid: ChainIdMismatch(Mismatch { expected: ChainId("00000000-0000-0000-0000-000000000000"), actual: ChainId("0") })

$ target/debug/iroha --config configs/swarm/client.toml domain list all
[
  {
    "id": "genesis",
    "accounts": {
      "ed01204164BF554923ECE1FD412D241036D863A6AE430476C898248B8237D77534CFC4@genesis": {
        "id": "ed01204164BF554923ECE1FD412D241036D863A6AE430476C898248B8237D77534CFC4@genesis",
        "assets": {},
        "metadata": {}
      }
    },
    "asset_definitions": {},
    "asset_total_quantities": {},
    "logo": null,
    "metadata": {},
    "owned_by": "ed01204164BF554923ECE1FD412D241036D863A6AE430476C898248B8237D77534CFC4@genesis"
  }
]
```

Reply:
This got me thinking, if we have an initial state with no committed transactions, why not replace it with the state right after the genesis block?

Reply:
Changed chain id in swarm from `00000000-0000-0000-0000-000000000000` to `0`

> This got me thinking, if we have an initial state with no committed transactions, why not replace it with the state right after the genesis block?

I am not fully understand. In order to get state after the genesis block we need to execute instructions in genesis block one-by-one, validating each instruction, etc?

Reply:
What is the point of changing to 0? I think the previous literal was easy to find and refactor in the future

I mean, we did not need any instructions for the genesis domain and account registered, which means we already have an assumption on the initial state across peers. Also, the genesis transaction goes through no validations. If so, why not have wonderland, alice, executor, chain ID, etc. as an configurable initial state?

Reply:
open a separate issue. Let's get this merged

Reply:
#4685